types/options.go

```diff
@@ -288,15 +289,20 @@ func ReloadConfigurationFile(configFile string, storeOptions *StoreOptions) {
 	if storeOptions.GraphDriverName == "" {
 		logrus.Errorf("The storage 'driver' option must be set in %s, guarantee proper operation.", configFile)
```

Comment:
Hey, that reminds me: the example config we provide in this repository triggers this message, but it is often used as-is by third-party packaging, because why wouldn't it?

Reply:
Would you want to change our default in storage.conf to "overlay", so that this is not an issue?

Reply:
That should work, I think.